docs/docs/how-to/previews-deploys-hosting/creating-an-adapter.md

```diff
@@ -146,6 +146,7 @@ The `config` hook has to return an object with the following keys:
   - `pathPrefix` (Optional): If `false`, Gatsby will fail the build if user tries to use pathPrefix
   - `trailingSlash` (Optional): Provide an array of supported [`trailingSlash`](/docs/reference/config-files/gatsby-config/#trailingslash) options, e.g. `['always']`
 - `pluginsToDisable` (Optional): Provide an array of plugin names that should be disabled when adapter is used. Purpose of this is to disable any potential plugins that serve similar role as adapter that would cause conflicts when both plugin and adapter is used at the same time.
+- `imageCDNUrlGeneratorModulePath` (Optional): Specifies the absolute path to a CommonJS module that exports a function to generate CDN URLs for images. This function, matching the `ImageCdnUrlGeneratorFn` type signature, takes `ImageCdnSourceImage` and `ImageCdnTransformArgs` as arguments to create optimized CDN URLs for image assets. It is particularly useful for adapting image paths to different CDN providers to optimize image delivery.
```

Comment:
need to add `fileCDNUrlGeneratorModulePath` and specify gatsby version it first will be available in (likely `gatsby@5.13.0`)